plugins/omkafka/omkafka.c

```diff
@@ -743,13 +786,156 @@ deliveryCallback(rd_kafka_t __attribute__((unused)) *rk,
 	} else {
 		DBGPRINTF("omkafka: kafka delivery SUCCESS on msg '%.*s'\n", (int)(rkmessage->len-1),
 			(char*)rkmessage->payload);
+		STATSCOUNTER_INC(ctrKafkaAck, mutCtrKafkaAck);
 	}
 finalize_it:
 	if(iRet != RS_RET_OK) {
 		DBGPRINTF("omkafka: deliveryCallback returned failure %d\n", iRet);
 	}
 }
 
+/**
+ * This function looks for a json object that corresponds to the
+ * passed name and returns it is found. Otherwise returns NULL.
+ * It will be used for processing stats callback json object.
+ */
+static struct fjson_object * get_object(struct fjson_object *fj_obj,
+		const char * name) {
+  struct fjson_object_iterator it = fjson_object_iter_begin(fj_obj);
```

Comment:
I guess you missed some indention here - sorry to be a pest

Reply:
sorry, missed that in first round. I'll update this and check others as well.